biomejs/biome
# chore: use the same `Prettier` version in Biome

## Summary

Some places still used old Prettier version, so all places use the same Prettier version `v3.1.0`.





## Test Plan

- Check if CI should pass
- Check if `node run.js` works in `benchmark` dir
- Check if `node prepare_tests.js <Prettier Repo>` works in `crates/biome_js_formatter/tests/specs/prettier`
- Check if `pnpm start` works in `website`
